scipy/special/add_newdocs.py

```diff
@@ -4407,22 +4407,218 @@ def add_newdoc(place, name, doc):
     """
     kolmogi(p)
 
-    Inverse function to kolmogorov
+    Inverse Survival Function of Kolmogorov distribution
 
+    It is the inverse function to `kolmogorov`.
     Returns y such that ``kolmogorov(y) == p``.
+
+    Parameters
+    ----------
+    p : float array_like
+        Probability
+
+    Returns
+    -------
+    float
+        The value(s) of kolmogi(p)
+
+    See Also
+    --------
+    kolmogorov : The Surival Function for the distribution
+    kolmogci : The Inverse CDF for the distribution
+    scipy.stats.kstwobign : Provides the functionality as a continuous distribution
+    smirnov, smirnovi, smirnovc, smirnovci, smirnovp : Functions for the one-sided distribution
+
+    Examples
+    --------
+    >>> from scipy.special import kolmogi
+    >>> kolmogi([0, 0.1, 0.25, 0.5, 0.75, 0.9, 1.0])
+    array([        inf,  1.22384787,  1.01918472,  0.82757356,  0.67644769,
+            0.57117327,  0.        ])
+
     """)
 
 add_newdoc("scipy.special", "kolmogorov",
-    """
+    r"""
     kolmogorov(y)
 
-    Complementary cumulative distribution function of Kolmogorov distribution
+    Complementary cumulative distribution (Survival Function) function of Kolmogorov distribution
 
     Returns the complementary cumulative distribution function of
-    Kolmogorov's limiting distribution (Kn* for large n) of a
+    Kolmogorov's limiting distribution (``D_n*\sqrt(n)`` for large n) of a
     two-sided test for equality between an empirical and a theoretical
     distribution. It is equal to the (limit as n->infinity of the)
     probability that sqrt(n) * max absolute deviation > y.
+
+    Parameters
+    ----------
+    y : float array_like
+      Absolute deviation between the ECDF and the CDF, multiplied by sqrt(n)
+
+    Returns
+    -------
+    float
+        The value(s) of kolmogorov(y)
+
+    See Also
+    --------
+    kolmogc : The CDF for the distribution
+    kolmogi : The Inverse Survival Function for the distribution
+    kolmogp : The derivative of the Survival Function for the distribution.
+    scipy.stats.kstwobign : Provides the functionality as a continuous distribution
+    smirnov, smirnovi, smirnovc, smirnovci, smirnovp : Functions for the one-sided distribution
+
+    Examples
+    --------
+    Show the probability of a gap at least as big as 0, 0.5 and 1.0.
+
+    >>> from scipy.special import kolmogorov, kolmogc
+    >>> kolmogorov([0, 0.5, 1.0])
+    array([ 1.        ,  0.96394524,  0.26999967])
+
+    Compare a sample of size 1000 drawn from a Laplace(0, 1) distribution against
```

Comment:
At first glance, I found this example confusing. I guess it is an illustration that the KS test rejects the hypothesis that the sample x is drawn from a normal distribution by computing the test statistic and then computing the p-value. I suggest to add a few more comments, maybe one could even add a line with kstest(x, "norm") which gives KstestResult(statistic=0.05828560005373079, pvalue=0.0021467598952917832) to confirm the result.